examples/cpp_parse_load_save/README.md

```diff
@@ -0,0 +1,79 @@
+# Parsing, Loading, and Saving MIGraphX Programs
+
+## Description
+This examples demonstrates how to parse, load, and save a graph program using the MIGraphX C++ API. 
+
+## Parsing
+Computation graphs that have been saved in a compatible serialized format, such as [ONNX](https://onnx.ai/get-started.html), can be read in by MIGraphX to create a runable program. 
+
+```
+migraphx::program p;
+unsigned batch = 1; //Or read in as argument
+migraphx::onnx_options options;
+options.set_default_dim_value(batch);
+p = parse_onnx(input_file, options);
+```
+
+## Saving
+An instantiated migraphx::program object can then be serialized to MessagePack (.msgpack) format and saved so that it can be loaded for future uses. 
+
+A program can be saved with either of the following:
+```
+migraphx::program p = ... <migraphx::program>;
+migraphx::save(p, output_file); 
+```
+
+```
+migraphx::program p = ... <migraphx::program>;
+migraphx_file_options options;
+options.format = "msgpack";
+migraphx::save(p, output_file, options);
+```
+
+## Loading
+Similarly, graphs that have been previously parsed, and possibly compiled, and then saved in either MessagePack or JSON format can be loaded at later time. 
+
+MessagePack is the default format, and can be loaded with either:
+```
+migraphx::program p;
+p = migraphx::load(input_file);
+```
+
+```
+migraphx::program p;
+migraphx_file_options options;
+options.format = "msgpack";
+p = migraphx::load(input_file, options);
+```
+To load a program that has been saved in JSON format:
+```
+migraphx::program p;
+migraphx_file_options options;
+options.format = "json";
+p = migraphx::load(input_file, options);
+```
+
+
+## Running the Example
+The provided example [`parse_load_save.cpp`](./parse_load_save.cpp) has these features implemented to allow for comparing outputs. 
+
+Running this example requires that MIGraphX has already been compiled and build on your system. If you have not yet done so, please refer to the instructions found [here](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/README.md).
```

Comment:
We should assume for the examples that migraphx is already installed.

Reply:
Removed